src/main/java/commission/entity/Sale.java

```diff
@@ -1,11 +1,9 @@
 package commission.entity;
 
-
-import org.springframework.boot.autoconfigure.domain.EntityScan;
-
-@EntityScan
  public record Sale (long id, double price, double percentage, double commission) {
 
-  public Sale{};
-
-}
+  @Override
```

Comment:
This class doesn't inherit from a class that has a commission method so the `@Override` annotation is not correct here.